Parsers/VSISDMI.cs

```diff
@@ -1666,14 +1656,19 @@ public VSISDMI()
                                 Name = "MMI_Q_LEVEL_NTC_ID",
                                 BitFieldType = BitFieldType.Bool,
                                 Length = 1,
-                                Comment = "Qualifier for the variable MMI_M_LEVEL_NTC_ID"
+                                Comment = "Qualifier for the variable MMI_M_LEVEL_NTC_ID",
```

Comment:
As discussed on skype, Bool does not match with "0" or "1"

Reply:
Ok, I will change it to UInt8